src/mediarithmics/plugins/common/BasePlugin.ts

```diff
@@ -35,7 +36,7 @@ import {
 
 import { Index, Option, flatMap, obfuscateString } from '../../utils';
 import { normalizeArray } from '../../utils/Normalizer';
-import { DataListResponse, Compartment } from "../../";
+import { DataListResponse, Compartment, asAssetFolderProperty } from "../../";
```

Comment:
Can you move `asAssetFolderProperty` in the import statement above (so that it's with the other ones)?